src/plugins/ini/testmod_ini.c

```diff
@@ -70,14 +70,18 @@ static void test_plainIniWrite(char *fileName)
 			keyNew ("user/tests/ini-write/nosectionkey",
 					KEY_VALUE, "nosectionvalue",
 					KEY_END),
-			keyNew ("user/tests/ini-write/section1", KEY_DIR, KEY_END),
+			keyNew ("user/tests/ini-write/section1",
+					KEY_BINARY,
```

Comment:
As far as I can tell the KEY_BINARY flag is already part of the keyNew documentation and the feature is also tested in testabi_key.c